templates/mongodb-atlas-main.template.yaml

```diff
@@ -176,20 +186,42 @@ Parameters:
     Description: MongoDB Atlas Database User  Name.
     Type: String
     Default: "testUser"
+  ServerlessProviderName:
+    Type: String
+    Description: Human-readable label that identifies the cloud service provider. Used only on the Serverless Instance
+    Default: "SERVERLESS"
+  ServerlessTerminationProtectionEnabled: 
+    Type: String
+    Description: Flag that indicates whether termination protection is enabled on the serverless instance. If set to true MongoDB Cloud won't delete the serverless instance. If set to false MongoDB cloud will delete the serverless instance.
+    ConstraintDescription: boolean
+    AllowedValues: 
+      - "true"
+      - "false"
+    Default: "false"
+  ServerlessContinuousBackupEnabled: 
+    Type: String
+    Description: Flag that indicates whether the serverless instances uses Serverless Continuous Backup. If this parameter is false the serverless instance uses Basic Backup. | Option | Description | |---|---| | Serverless Continuous Backup | Atlas takes incremental snapshots of the data in your serverless instance every six hours and lets you restore the data from a selected point in time within the last 72 hours. Atlas also takes daily snapshots and retains these daily snapshots for 35 days. To learn more see Serverless Instance Costs. | | Basic Backup | Atlas takes incremental snapshots of the data in your serverless instance every six hours and retains only the two most recent snapshots. You can use this option for free. 
+    ConstraintDescription: boolean
+    AllowedValues: 
+      - "true"
+      - "false"
+    Default: "true"
   DatabasePassword:
     Description: MongoDB Atlas Database User Password.
     Type: String
     NoEcho: true
 Conditions:
   UsingDefaultBucket: !Equals [!Ref QSS3BucketName, 'aws-quickstart']
   ActivateResources: !Equals [!Ref ActivateMongoDBResources, 'Yes']
+  CreateCluster: !Equals [!Ref InstanceType, 'Dedicated Cluster']
+  CreateServerless: !Equals [!Ref InstanceType, 'Serverless Instance']
 Resources:
   ActivateAtlasResources:
     Condition: ActivateResources
     Type: AWS::CloudFormation::Stack
     Properties:
       TemplateURL: !Sub
-        - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/activate-mongodb-atlas-resources.template.yaml'
+        - 'https://mongodbtemplate.s3.us-west-2.amazonaws.com/stacktemplate/activate-mongodb-atlas-resources.template.yaml'
```

Comment:
here you left your s3, can we make it to the original?

Reply:
updated the default URL